src/bitcoin-chainstate.cpp

```diff
@@ -18,6 +18,7 @@
 #include <chainparams.h>
 #include <consensus/validation.h>
 #include <core_io.h>
+#include <cs_main.h>
```

Comment:
I think it makes more sense to have `#include <validation.h>` imply that `cs_main` is available -- in the short term it reduces the diff here, but even in the long term, I think we can expect `cs_main` to become a validation specific lock.

Here's what that would look like: https://github.com/ajtowns/bitcoin/commit/f4d29cf1fe2edeaf2892fe69502fd7dcab9f5a5d -- it just takes the current patch, drops the redundant cs_main includes and adds an IWYU pragma to prevent iwyu from suggesting they get added back.